docs/website/docs/plus/getting-started/transformations.md

```diff
@@ -0,0 +1,446 @@
+office_table = star_dataset.offices
+
+joined_table = fact_table.join(office_table, fact_table.office_id == office_table.id)
+print(joined_table.group_by(office_table.city).aggregate(total_payments=joined_table.amount.sum()).arrow())
+
+
+```
+
+:::tip
+For simply copying table data, you can also use the sync source demonstrated below. Also note that in future versions of the dlt+ transformation framework, there will be a template generator to create a star schema from a given dataset to give you a starting point.
+:::
+
+
+## Lineage
+
+dlt+ transformations contain a lineage engine that can trace the origin of columns resulting from transformations. You may have noticed that we added a custom hint to the name column in the employees table at the beginning of the page. This hint is a custom hint that we decided to add to all columns containing very sensitive data. Ideally, we would like to know which columns in a result are derived from columns containing sensitive data. dlt+ lineage will do just that for you. Let's run an aggregated join query into our warehouse again, but this time we will not drop the name column:
+
+```python
+
+@dlt_plus.transform(transformation_type="python")
+def payment_per_employee(dataset: SupportsReadableDataset):
+    employee_tables = dataset.employees
+    payroll_table = dataset.payroll
+    joined_table = employee_tables.join(payroll_table, employee_tables.id == payroll_table.employee_id)
+    return joined_table.group_by(joined_table.employee_id, joined_table.name).aggregate(total_payments=joined_table.amount.sum())
+
+# run the transformation
+info = warehouse_pipeline.run(payment_per_employee(company_dataset))
+print(info)
+
+```
+
+You can now inspect the schema of the warehouse and see that the name column of the aggregated table is also marked with our hint: `uv run dlt pipeline warehouse_pipeline schema`. dlt+ uses the same mechanism to derive precision and scale information for columns.
+
+You can control the lineage mode with the `lineage_mode` parameter on the dlt.transform decorator, which can be set to `strict`, `best_effort`, or `disabled`. The default is `best_effort`, which will try to trace the lineage for each column and set the derived hints but will not fail if this is not possible. SQL-based transformations that introduce non-traceable columns will require you to supply column type hints for any columns that could not be traced. For Python-based transformations, the column type can be derived from the incoming data. In `strict` mode, any transformation that introduces non-traceable columns will fail with a useful error. This mode is recommended if you must ensure which columns may contain sensitive data and need to prevent them from being unmarked. You can also disable lineage for a specific transformation by setting the `lineage_mode` to `disabled`. In this mode, the full schema will be derived from the incoming data and no additional hints will be added to the resulting dataset.
```

Comment:
this section is a bit long and would profit from being turned into some kind of a list or table or the like.